docs(stepper): Make CDK stepper example accessible in dark mode.

walvekarnikhil opened the pull request:
Removed hard-coded colors and replaced with inherit colors.
Used font-weight to show active step.

Fixes #17152

walvekarnikhil commented:
Do we have flaky tests? 
Same tests failed for https://app.circleci.com/pipelines/github/angular/components/10541/workflows/2f4623fd-1b18-453d-b244-00b59863eca3/jobs/165832

walvekarnikhil commented:
Here are the screenshots for the CDK stepper examples.
Light mode:
<img width="1644" alt="Screenshot 2020-07-15 at 10 05 32 PM" src="https://user-images.githubusercontent.com/5762367/87741617-93009480-c802-11ea-8de9-13dc38aac783.png">

Dark mode:
<img width="1635" alt="Screenshot 2020-07-15 at 10 03 08 PM" src="https://user-images.githubusercontent.com/5762367/87741628-98f67580-c802-11ea-8052-c779d83f9fa0.png">

walvekarnikhil commented:
It seems we have a flaky test, it passed after a fake commit.

crisbeto commented on src/components-examples/cdk/stepper/cdk-custom-stepper-without-form/example-custom-stepper.css:
```diff
@@ -1,7 +1,8 @@
+  border: 1px solid;
   padding: 10px;
   margin: 10px;
+  color: inherit;
```
I think all of these `color: inherit` uses are redundant. Doesn't `color` default to `inherit`?

walvekarnikhil replied:
Fixed, kept color: inherit for nav buttons since those were shown in black for dark mode.

crisbeto commented on src/components-examples/cdk/stepper/cdk-custom-stepper-without-form/example-custom-stepper.css:
```diff
@@ -10,24 +11,22 @@
 }
 
 .example-step.example-active {
-  color: blue;
```
Maybe we can pick a color that stands out both on light and dark backgrounds? I believe the point of this example is to make it stand out to illustrate a point, rather than be pretty.

walvekarnikhil replied:
Yes right, we discussed that point but there is an issue with maintaining text contrast for both light and dark mode. So instead of using a neutral color, we ended using the text colors.

crisbeto replied:
Alright, the `font-weight` that you replaced it with should be fine as well.

walvekarnikhil commented:
@crisbeto , fixed the commit message, earlier I had just modified PR title my bad. 
Do I need to squash other commits also or those are done when we merge the PR?

crisbeto commented:
I think you still need to change your PR title, because that's what will be used when the caretaker squashes the commits.

walvekarnikhil commented:
> I think you still need to change your PR title, because that's what will be used when the caretaker squashes the commits.

Title updated.
